src/library/hiddentablemodel.cpp

```diff
@@ -22,13 +21,16 @@ void HiddenTableModel::setTableModel(int id) {
     QStringList columns;
     columns << "library." + LIBRARYTABLE_ID;
     QString filter("mixxx_deleted=1");
-    query.prepare("CREATE TEMPORARY VIEW IF NOT EXISTS " + tableName + " AS "
-                  "SELECT "
-                  + columns.join(",") +
-                  " FROM library "
-                  "INNER JOIN track_locations "
-                  "ON library.location=track_locations.id "
-                  "WHERE " + filter);
+    query.prepare(
+            "CREATE TEMPORARY VIEW IF NOT EXISTS " + tableName +
+            " AS "
+            "SELECT " +
```

Comment:
This looks horrible imo, if someone has a way to write it that fits clang-format, you are welcome!

Reply:
The nice thing with clang-format is that we can stop arguing about horrible or nice. 

It contains the common denominator that works for all. Of case you can find places where clang-format "fails" but that is OK by definition.

You may try to trick it but that is absolutely not necessary. Just go ahead.

Reply:
Inspired by @Holzhaus I changed it to this, does that make sense? 63bf05a

Reply:
AFAIK bindValue only works for values, not table names, column lists or boolean expressions.